src/transformers/integrations/integration_utils.py

```diff
@@ -364,10 +376,11 @@ def dynamic_modules_import_trainable(*args, **kwargs):
     return best_run
 
 
-def run_hp_search_sigopt(trainer, n_trials: int, direction: str, **kwargs) -> BestRun:
+def run_hp_search_sigopt(trainer, n_trials: int, direction: str, use_best_model: bool = False, **kwargs) -> BestRun:
     import sigopt
 
     if trainer.args.process_index == 0:
+        print("In master of sigopt")
```

Comment:
If we ever want to merge this PR, this should be removed.

Reply:
Yes, thanks for catching this.